app/src/main/java/com/github/braillesystems/learnbraille/ui/screens/browser/MarkerViewFragment.kt

```diff
@@ -18,9 +23,63 @@ import com.github.braillesystems.learnbraille.ui.views.BrailleDotsViewMode
 import com.github.braillesystems.learnbraille.ui.views.display
 import com.github.braillesystems.learnbraille.ui.views.dotsState
 import com.github.braillesystems.learnbraille.utils.*
+import com.karlotoy.perfectune.instance.PerfectTune
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+enum class NoteDuration(
+    val titleStrId: Int,
+    val dot3: Boolean,
+    val dot6: Boolean,
+    val valueMillis: Long
+) {
+    EIGHTH(titleStrId = R.string.note_duration_8th, dot3 = false, dot6 = false, valueMillis = 125),
+    QUARTER(titleStrId = R.string.note_duration_4th, dot3 = false, dot6 = true, valueMillis = 250),
+    HALF(titleStrId = R.string.note_duration_half, dot3 = true, dot6 = false, valueMillis = 500),
+    FULL(titleStrId = R.string.note_duration_full, dot3 = true, dot6 = true, valueMillis = 1000);
+
+    fun modifiedNote(note: BrailleDots): BrailleDots {
```

Comment:
Maybe it is better to add docs here: purpose of this function is not clear

Reply:
Added a single line description of this method